src/localization/strings.ts

```diff
@@ -69,6 +69,12 @@ export interface Localized extends LocalizedStringsMethods {
       formScreen: {
         title: string;
         description: string;
+        form: {
+          form: string;
+          latest: string;
+          new: string;
+          notification: string;
+        }
```

Comment:
```suggestion
        form: {
          form: string;
          latestReply: string;
          newReply: string;
          notProcessed: string;
        }
```Maybe something like that for the naming